Skrub Add Estimated Memory Usage to TableReport (#2153)
skrub-data/skrub: CHANGES.rst

  :pr:`2096` by :user:`Ayesha Siddiqua <siddiqua-tamk>`.
- The :class:`TableReport` can now be exported in markdown format with ``.markdown``.
  :pr:`2048` by :user:`Riccardo Cappuzzo <rcap107>`.

> - The :class:`TableReport` can now be exported the estimated memory usage in TableReport when display data.

Reviewer: ```suggestion
- The :class:`TableReport` can now display the estimated memory usage of
  the data it is applied to.
```
